<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KORAT" ignored="1" name="Default test(failed)(failed)" tests="4" failures="1" timestamp="2022-11-16T22:55:24 EST" time="0.101" errors="0">
  <testcase name="webMethod" time="0.027" classname="com.practise.web1">
    <skipped/>
  </testcase> <!-- webMethod -->
  <testcase name="webMethod" time="0.005" classname="com.practise.web1">
    <skipped/>
  </testcase> <!-- webMethod -->
  <testcase name="webMethod" time="0.002" classname="com.practise.web1">
    <skipped/>
  </testcase> <!-- webMethod -->
  <testcase name="webMethod" time="0.0" classname="com.practise.web1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.practise.web1.webMethod(web1.java:29)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- webMethod -->
  <testcase name="phoneMethod" time="0.0" classname="com.practise.web1">
    <ignored/>
  </testcase> <!-- phoneMethod -->
</testsuite> <!-- Default test(failed)(failed) -->
